coordinator/auth-api/pom.xml

```diff
@@ -1,14 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <!-- inherit common settings -->
+  <!-- Parent -->
   <parent>
     <groupId>io.stargate</groupId>
     <artifactId>stargate</artifactId>
     <version>2.0.10-SNAPSHOT</version>
   </parent>
+  <!-- Artifact props -->
   <groupId>io.stargate.auth.api</groupId>
   <artifactId>auth-api</artifactId>
+  <name>Stargate - Coordinator - Auth API</name>
```

Comment:
was generally adapting the names, so we can get better outputs in console and IDE:


![image](https://user-images.githubusercontent.com/10600041/223433660-0a18197b-830a-4fe1-9ee5-8483450c7de3.png)

```
[INFO] Stargate - Coordinator Parent pom .................. SUCCESS [  0.077 s]
[INFO] Stargate - Coordinator - Core ...................... SUCCESS [  0.538 s]
[INFO] Stargate - Coordinator - Starter ................... SUCCESS [  0.484 s]
[INFO] Stargate - Coordinator - Persistence API ........... SUCCESS [  0.384 s]
[INFO] Stargate - Coordinator - Persistence commons ....... SUCCESS [  0.069 s]
[INFO] Stargate - Coordinator - Persistence test .......... SUCCESS [  0.146 s]
[INFO] Stargate - Coordinator - AuthN/Z ................... SUCCESS [  0.027 s]
[INFO] Stargate - Coordinator - Persistence C* 3.11 ....... SUCCESS [  0.144 s]
[INFO] Stargate - Coordinator - Persistence C* 4.0 ........ SUCCESS [  0.118 s]
[INFO] Stargate - Coordinator - Bridge proto .............. SUCCESS [  0.463 s]
[INFO] Stargate - Coordinator - Bridge .................... SUCCESS [  0.119 s]
[INFO] Stargate - Coordinator - CQL ....................... SUCCESS [  0.120 s]
[INFO] Stargate - Coordinator - Metrics Jersey ............ SUCCESS [  0.055 s]
[INFO] Stargate - Coordinator - Graphql API (deprecated) .. SUCCESS [  0.220 s]
[INFO] Stargate - Coordinator - REST API (deprecated) ..... SUCCESS [  0.198 s]
[INFO] Stargate - Coordinator - Auth API .................. SUCCESS [  0.024 s]
[INFO] Stargate - Coordinator - Auth Table-based service .. SUCCESS [  0.017 s]
[INFO] Stargate - Coordinator - Auth JWT service .......... SUCCESS [  0.040 s]
[INFO] Stargate - Coordinator - Health checker ............ SUCCESS [  0.022 s]
[INFO] Stargate - Coordinator - Config store API .......... SUCCESS [  0.020 s]
[INFO] Stargate - Coordinator - Config store YAML ......... SUCCESS [  0.012 s]
[INFO] Stargate - Coordinator - Rate limiting ............. SUCCESS [  0.006 s]
[INFO] Stargate - Coordinator - gRPC proto ................ SUCCESS [  0.100 s]
[INFO] Stargate - Coordinator - gRPC ...................... SUCCESS [  0.081 s]
[INFO] Stargate - Coordinator - gRPC examples ............. SUCCESS [  0.017 s]
[INFO] Stargate - Coordinator - Testing services .......... SUCCESS [  0.005 s]
[INFO] Stargate - Coordinator - Testing ................... SUCCESS [  0.167 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  4.578 s
[INFO] Finished at: 2023-03-07T13:56:06+01:00
[INFO] ------------------------------------------------------------------------

```